web/src/lib/components/admin-page/settings/machine-learning-settings/machine-learning-settings.svelte

```diff
@@ -77,6 +77,40 @@
         </div>
       </SettingAccordion>
 
+      <SettingAccordion
+        key="duplicate-detection"
+        title="Duplicate Detection"
+        subtitle="Use CLIP embeddings to find very similar images that are likely duplicates"
+      >
+        <div class="ml-4 mt-4 flex flex-col gap-4">
+          <SettingSwitch
+            id="enable-duplicate-detection"
+            title="ENABLED"
+            subtitle="If disabled, images will not be checked to see if they are duplicates."
```

Comment:
```suggestion
            subtitle="If disabled, exactly identical assets will still be de-duplicated."
```
Clarifies that checksum deduplication will still work.